roachtest: tpcc/nodes=3/w=max failed

SHA: https://github.com/cockroachdb/cockroach/commits/032c4980720abc1bdd71e4428e4111e6e6383297

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=tpcc/nodes=3/w=max PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1158877&tab=buildLog

```
The test failed on master:
	cluster.go:1226,tpcc.go:126,cluster.go:1564,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1158877-tpcc-nodes-3-w-max:4 -- ./workload run tpcc --warehouses=1350 --histograms=logs/stats.json  --ramp=5m0s --duration=2h0m0s {pgurl:1-3} returned:
		stderr:
		
		stdout:
		 0           20.1           28.8    369.1    536.9    536.9    536.9 delivery
		1h54m59s        0          294.5          287.1    302.0    536.9    637.5    671.1 newOrder
		1h54m59s        0           24.1           28.8     28.3     79.7     96.5     96.5 orderStatus
		1h54m59s        0          283.4          287.6    234.9    436.2    520.1   1073.7 payment
		1h54m59s        0           27.1           28.8    151.0    335.5    604.0    604.0 stockLevel
		 1h55m0s        0           37.0           28.8    243.3    402.7    486.5    486.5 delivery
		 1h55m0s        0          290.2          287.1    159.4    419.4    486.5    536.9 newOrder
		 1h55m0s        0           25.0           28.8     22.0     48.2     62.9     62.9 orderStatus
		 1h55m0s        0          321.3          287.6    125.8    318.8    369.1    503.3 payment
		 1h55m0s        0           27.0           28.8    184.5    251.7    436.2    436.2 stockLevel
		Error: error in delivery: ERROR: no inbound stream connection (SQLSTATE XX000)
		Error:  exit status 1
		: exit status 1
	cluster.go:1585,tpcc.go:136,tpcc.go:160,test.go:1211: Goexit() was called

```

The `exit status 255` failure is coming from ssh. With `-v` added to the ssh command line, we see that it's a "broken pipe"

```
--- FAIL: tpcc/nodes=3/w=max (5070.79s)
	cluster.go:1293,tpcc.go:119,tpcc.go:121,tpcc.go:165,test.go:1223: /Users/bdarnell/cockroach/bin/roachprod run ben-1553980169-tpcc-nodes-3-w-max:4 -- ./workload fixtures load tpcc --checks=true --warehouses=2000 {pgurl:1} returned:
		stderr:
		ebug1: Authentication succeeded (publickey).
		Authenticated to 35.185.49.255 ([35.185.49.255]:22).
		debug1: channel 0: new [client-session]
		debug1: Requesting no-more-sessions@openssh.com
		debug1: Entering interactive session.
		debug1: pledge: network
		debug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0
		debug1: Sending environment.
		debug1: Sending env LANG = en_US.UTF-8
		debug1: Sending command: export ROACHPROD=4 GOTRACEBACK=crash && bash -c "./workload fixtures load tpcc --checks=true --warehouses=2000 'postgres://root@10.142.0.32:26257?sslmode=disable'"
		I190330 21:11:51.954354 1 ccl/workloadccl/cliccl/fixtures.go:296  starting load of 9 tables
		I190330 21:14:36.156765 150 ccl/workloadccl/fixture.go:600  loaded district (2m44s, 20000 rows, 0 index entries, 2.0 MiB)
		I190330 21:17:24.120448 149 ccl/workloadccl/fixture.go:600  loaded warehouse (5m32s, 2000 rows, 0 index entries, 106 KiB)
		packet_write_wait: Connection to 35.185.49.255 port 22: Broken pipe
		Error:  exit status 255
		
		stdout:
		: exit status 1
FAIL
```

This is the error that is expected if an intermediary network device forgets about the connection due to inactivity. However, we're running with ServerAliveInterval=60 to prevent the connection from being completely idle (unless there's something in the path that has a 60s timeout and we need to send our pings every 55s to reliably keep the connection alive. But 60s would be very short for a timeout at this level). We see this in a variety of network conditions, including teamcity where everything should be staying within the GCP network. 

I would expect problems on the host we're sshing to to result in "connection reset" errors, not "broken pipe". But I can't rule it out; there may be something that could cause this failure mode instead. 

Within minutes of first boot we're getting (and rejecting) brute force ssh attacks. Shouldn't make a difference; just an observation:

    Mar 31 14:13:01 ben-1554041301-tpcc-nodes-3-w-max-0004 sshd[3435]: error: maximum authentication attempts exceeded for root from 42.61.24.202 port 35112 ssh2 [preauth]
    Mar 31 14:13:01 ben-1554041301-tpcc-nodes-3-w-max-0004 sshd[3435]: Disconnecting: Too many authentication failures [preauth]


In any case, the cockroach processes seem fine even after the test has failed with `exit status 255`. They don't crash, and they keep processing the import. So this subset of failures appears to just be an issue of test automation and doesn't indicate a problem in cockroach.

--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -156,9 +156,9 @@ func registerTPCC(r *registry) {
 			var warehouses int
 			switch cloud {
 			case "gce":
-				warehouses = 1350
+				warehouses = 1250
 			case "aws":
-				warehouses = 2300
+				warehouses = 2100
 			default:
 				t.Fatalf("unknown cloud: %q", cloud)
 			}

